packages/commonwealth/shared/utils.ts

```diff
@@ -71,7 +71,7 @@ export const requiresTypeSlug = (type: ProposalType): boolean => {
 };
 
 /* eslint-disable */
-export const getThreadUrl = (type, thread, comment?) => {
+export const getThreadUrl = (thread, comment?) => {
```

Comment:
there are 6 occurrences of getThreadUrl() and 2 of those are passing 'discussion' as a first param, so I believe that they will be broken after this change. 

![image](https://user-images.githubusercontent.com/14819225/235124345-b657f757-a3ac-4187-8b38-54f2cd0649e6.png)